src/types/index.ts

```diff
@@ -615,6 +616,7 @@ export type ApiPayment = {
   receiver_wallet: ApiPaymentReceiverWallet;
   created_at: string;
   updated_at: string;
+  external_payment_id: string;
```

Comment:
Same as above. It might be helpful to indicate that we might not always have the payment ID.

```suggestion
  external_payment_id?: string;
```